examples/elaboratePoint2KalmanFilter.cpp

```diff
@@ -55,17 +55,21 @@ int main() {
 
   // Create new state variable
   Symbol x0('x',0);
-  ordering->insert(x0, 0);
+  ordering->push_back(x0);
```

Comment:
Maybe use X(k) everywhere.
```
using symbol_shorthand::X;
```

Reply:
No, I mean: kill x0, x1, x2 variables. Use X(k) in-place everywhere. Slightly less efficient but good practice for copy/paste in other code.